manifest/manifest.go

```diff
@@ -279,6 +293,58 @@ func stringVal(yamlMap map[string]interface{}, key string, errs *[]error) *strin
 	return &result
 }
 
+func bytesVal(yamlMap map[string]interface{}, key string, errs *[]error) *int64 {
+	yamlVal := yamlMap[key]
+	if yamlVal == nil {
+		return nil
+	}
+
+	stringVal := coerceToString(yamlVal)
+	value, err := formatters.ToMegabytes(stringVal)
+	if err != nil {
+		*errs = append(*errs, fmt.Errorf("Invalid value for '{{.PropertyName}}': {{.StringVal}}\n{{.Error}}",
+			map[string]interface{}{
+				"PropertyName": key,
+				"Error":        err.Error(),
+				"StringVal":    stringVal,
+			}))
+		return nil
+	}
+	return &value
+}
+
+func intVal(yamlMap map[string]interface{}, key string, errs *[]error) *int {
```

Comment:
That sounds like a good idea. We just copied the function from https://github.com/cloudfoundry/cli/blob/14f6dd00ab0332e00cc8da8735d46b731a599ae6/cf/manifest/manifest.go
We should add a test for "Applications" or "GetAppParams" in manifest_test.go where Memory, DiskQuota and InstanceCount are specifically tested